### PR TITLE
Add edgarlr/strapi-template-magazine template

### PR DESCRIPTION
**Pull request by edgarlr**
Add edgarlr/strapi-template-magazine to templates.yml

### CODE REVIEW

File: templates/templates.yml [hunk 1]
```diff
@@ -92,3 +92,24 @@
...
+    - Preview unpublished content – Articles, pages.
+    - Slug system
+    - 2 Contributor types – Featured, default.
+    - SEO and social media friendly
+    - Role based access controls
+  starters:
+    - https://github.com/edgarlr/magazine
+  command:
+    yarn: yarn create strapi-app my-app-name --template https://github.com/edgarlr/strapi-template-magazine
+    npm: npx create-strapi-app my-app-name --template https://github.com/edgarlr/strapi-template-magazine
```
Comment (markkaylor) [inline templates/templates.yml:115]: You can use a shortcut for the repo.  https://strapi.io/documentation/developer-docs/latest/setup-deployment-guides/installation/templates.html#using-a-template
```suggestion
    yarn: yarn create strapi-app my-app-name --template edgarlr/magazine
    npm: npx create-strapi-app my-app-name --template edgarlr/magazine
```

**Review by markkaylor — CHANGES_REQUESTED** [review]

**Comment by markkaylor** [comment]
If you could add seed data that would be awesome.  The process isn't really documented but you can see how it is done here: https://github.com/strapi/strapi-template-blog/blob/master/template/config/functions/bootstrap.js. Let me know if you have any questions.

**Comment by edgarlr** [comment]
@markkaylor Sure, I'll try to add seed data!

**Comment by edgarlr** [comment]
Hey @markkaylor, I just added the seed data! can you review it again?

**Pull request closed without merging**